enterprise/api/v1/serializers.py

```diff
@@ -954,17 +954,35 @@ def validate(self, data):  # pylint: disable=arguments-differ
         return data
 
 
+class LicensesInfoSerializer(serializers.Serializer):
+    """
+    Nested serializer class to allow for many license info dictionaries.
+    """
+    email = serializers.CharField(required=False)
+    course_run_key = serializers.CharField(required=False)
+    license_uuid = serializers.CharField(required=False)
+
+    def create(self, validated_data):
+        return validated_data
+
+    def validate(self, data):  # pylint: disable=arguments-differ
+        missing_fields = []
+        for key in self.fields.keys():
+            if not data.get(key):
+                missing_fields.append(key)
+
+        if missing_fields:
+            raise serializers.ValidationError('Found missing licenses info field(s): {}.'.format(missing_fields))
```

Comment:
`licenses info` -> `licenses_info` to match exact param name would be great

Reply:
also, 'found missing' seems confusing. Perhaps just "missing parameter `licenses_info` in request" or similar?